tests/communities/test_resources.py

```diff
@@ -689,6 +689,9 @@ def test_logo_max_content_length(
         data=BytesIO(logo_data),
     )
     assert res.status_code == 400
+    assert (
+        res.json["message"] == "Logo size limit exceeded. Limit: 1.00 MB Given: 4.00 MB"
```

Comment:
I'm going to look for other places that I could test this new function, but this seemed like a good spot to start. I'm not sure how thorough the tests needs to be.